roles/openshift_node_upgrade/tasks/main.yml

```diff
@@ -9,14 +9,35 @@
 # - openshift_release
 
 # tasks file for openshift_node_upgrade
+
+- name: Stop node and openvswitch services
+  service:
+    name: "{{ item }}"
+    state: stopped
+  with_items:
+  - "{{ openshift.common.service_type }}-node"
+  - openvswitch
+  failed_when: false
+
+- name: Stop additional containerized services
+  service:
+    name: "{{ item }}"
+    state: stopped
+  with_items:
+    - "{{ openshift.common.service_type }}-master"
+    - "{{ openshift.common.service_type }}-master-api"
+    - "{{ openshift.common.service_type }}-master-controllers"
```

Comment:
Given `master-controller` requires `master-api` [1], the `master-controller` should be stopped before the `master-api` (assuming all items are processed in the written order).

[1] https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_master/templates/docker-cluster/atomic-openshift-master-controllers.service.j2#L5